server/backend/env-docker-compose.env

```diff
@@ -1,5 +1,5 @@
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD=postgres
+POSTGRES_PASSWORD=test
 POSTGRES_HOST=localhost
```

Comment:
change to match the password in the docker-compose file in the root directory, for local Postgres setup

Reply:
No objection here but I thought normally people use default-user "postgres" with password also as "postgres" for easy to remember and use. 
"test" probably was intended for api-proxy user. But see what other people say.

Reply:
Makes sense to be consistent, but I can see Ian's point. Again just for local so I have no personal preference here. To make Ian happy could change both passwords to just be 'postgres' instead of 'test'.

Reply:
sure, I could update the docker-compose file to use password 'Postgres'